### PR TITLE
#1044: add plain text emails

### PR DESCRIPTION
**Pull request by samrichca**
## Ticket
- https://github.com/OHCRN/platform/issues/1044

## Description

This PR adds plain text emails for update password confirmation, reset password, and email verification. It also updates the i18n and HTML emails to facilitate plaintext emails.

### CODE REVIEW

**Review by shailikadakia-oicr — APPROVED** [review]
tested locally and the plain text looks great, thank you!

File: email/html/event-update_password.ftl [hunk 1]
```diff
@@ -2,19 +2,30 @@
...
 
 <@layout.emailLayout title=msg("eventUpdatePasswordSubject")>
   <#--  ENGLISH  -->
   <@body.emailBody_en title=msg("eventUpdatePassword_title_en")>
     <@text.emailText><b>${msg("eventUpdatePassword_welcome_en")}</b></@text.emailText>
-    <@text.emailText>${msg("eventUpdatePassword_text_en")?no_esc}</@text.emailText>
+    <@text.emailText>${msg("eventUpdatePassword_text1_en")?no_esc}</@text.emailText>
+		<@text.emailText>
+			${msg("eventUpdatePassword_text2_en")?no_esc} 
+			<@textLink.emailTextLink href=properties.ohcrnInfoEmail>${properties.ohcrnInfoEmail}</@textLink.emailTextLink>
```
Comment (mistryrn) [inline email/html/event-update_password.ftl:16]: For the HTML email I think we want the href to be prefixed with `mailto:` if it is an email address. Let's double-check the affected HTML emails and make sure those hrefs get the prefix, the plaintext version is good as you have it though

**Review by mistryrn — COMMENTED** [review]
Looks good! Just one suggestion for the affected HTML emails to retain their `mailto:` links

File: email/html/event-update_password.ftl (hunk 1 above)
Comment (samrichca) [reply email/html/event-update_password.ftl]: added mailto - if it doesn't work there's probably an email client configuration issue on your mac (found this out the hard way)

File: email/html/template/textLink.ftl [hunk 2]
```diff
@@ -1,8 +1,9 @@
 <#-- BODY - link in a text paragraph  -->
+<#--  `mailto:` can't be added to the target URL. for an email link, add mailto=true  -->
 
-<#macro emailTextLink href>
+<#macro emailTextLink href mailto>
```
Comment (mistryrn) [inline email/html/template/textLink.ftl:4]: Screenshot 2024-12-04 at 2 52 29 PM

Looks like this makes `mailto` a required prop whenever using this macro, causing the other email links to fail to compile (tried sending a verification email and got this error)

**Review by mistryrn — CHANGES_REQUESTED** [review]
Thanks for adding the `mailto:`, unfortunately this is breaking the other emails now 😅

File: email/html/template/textLink.ftl (hunk 2 above)
Comment (samrichca) [reply email/html/template/textLink.ftl]: sorry about that, updated

**Review by mistryrn — APPROVED** [review]
Thanks for the quick fix! Tested locally, plain text and HTML emails all working as expected 🎉

**Pull request merged**